commcare_connect/templates/tailwind/layouts/header.html

```diff
@@ -24,11 +28,11 @@ <h1 class="text-lg text-brand-deep-purple font-medium">{{ header_title|default:"
                 <!-- Organization Selector -->
                 <div x-show="open" x-transition.opacity x-cloak class="absolute top-12 right-0 min-w-56 bg-white shadow rounded-lg z-50">
                     <ul class="p-2 w-full">
-                        {% for membership in request.memberships %}
+                        {% for membership in request.user.memberships.all %}
```

Comment:
Why is this change needed?

Reply:
That change was for loading the Organization only when user had memberships, the `request.memberships` objecte evaluated to none when user had no memberships, but the `request.memberships` object also excludes the currently opened org from the dropdown, so i have replaced the code with a check and kept the old change here.
[ff35051](https://github.com/dimagi/commcare-connect/pull/617/commits/ff350514420d2d71255fd2ee9817fa57dd78276e)